CONTRIBUTING.md

```diff
@@ -1,21 +1,54 @@
 # Contributing
 
-We do not yet have a process to accept external contributions.
-If you are interested in contributing, you can let us know on our [Slack workspace](https://join.slack.com/t/tmltdev/shared_invite/zt-1bky0mh9v-vOB8azKAVoxmzJDUdWd5Wg) or by email at [support@tmlt.io](mailto:support@tmlt.io).
+We are happy to accept external contributions! 💖
+
+## Contact
+
+First, let us know what you would like to contribute. Feel free to:
+
+- report a bug or request a feature on [GitHub](https://github.com/opendp/tumult-analytics/issues);
+- send general queries to info@opendp.org, or email security@opendp.org if it is related to security;
+- ask any question on our [Slack][slack] instance.
+
+[slack]: https://join.slack.com/t/opendp/shared_invite/zt-1aca9bm7k-hG7olKz6CiGm8htI2lxE8w
 
 ## Local development
 
-We use [Poetry](https://python-poetry.org/) for dependency management during development.
-To work locally, install Poetry, and then install our dev dependencies by running `poetry install` from the root of this repository.
+### Installation
+
+We recommend using a Python virtual environment to do local development, for example using [pyenv](https://github.com/pyenv/pyenv): it is useful to make sure you're working with a supported Python version, and minimize interactions with your system Python environment.
```

Comment:
```suggestion
We recommend using the lowest supported python version for local development. Tools like [pyenv](https://github.com/pyenv/pyenv) may be useful for managing your python version.
```

pyenv doesn't actually manage virtual environments, despite the name. It just managed installed python versions. Poetry handles creating the virtual environments.

At some point (soon?) we probably want to switch these instructions to recommend `uv` instead (cc @tmager ), but I don't see a super easy way to substitute `uv` for `pyenv`.